core/src/main/scala/za/co/absa/spline/harvester/conf/YAMLConfiguration.scala

```diff
@@ -58,8 +58,10 @@ class YAMLConfiguration(yaml: String)
         case null => // ignore entries with `null` value
         case m: ju.Map[_, _] =>
           populateRecursively(dstKey, m.asInstanceOf[ju.Map[String, AnyRef]], dstMap)
+        case xs: ju.List[_] =>
+          dstMap.put(dstKey, xs)
```

Comment:
Shouldn't the list elements be also converted to String?

Reply:
The main reason why I decided not to convert the list items to string was the signature of the `Configuration.getList()` method, that returns a list of objects (not strings). Having an alternative method `getStringArray()` that returns arrays of strings I assumed that `getList` could be used to return a collection of typed values (if the underlying implementation supports one).